packages/app/src/layouts/default.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="h-screen overflow-hidden flex flex-row">
-    <main class="h-screen min-w-0 flex-1 border-t border-gray-200 lg:flex">
```

Comment:
This is probably going to conflict with my PR about layout and specs

Reply:
Yeah the expanded navbar isn't pushing the content right so that's for the best.